src/net/java/sip/communicator/impl/version/NightlyBuildID.java

```diff
@@ -27,5 +27,5 @@
     /**
      * The build ID.
      */
-    public static final String BUILD_ID="0.build.by.SVN";
+    public static final String BUILD_ID="1";
 }
```

Comment:
When invoking ant you just need to pass -Dlabel="1" and this is going to be set as build version.